lib/Doctrine/DBAL/Platforms/MySqlPlatform.php

```diff
@@ -594,6 +596,13 @@ public function getAlterTableSQL(TableDiff $diff)
                 continue;
             }
 
+            $type = get_class($column->getType());
+            if ($type !== StringType::class && $type !== TextType::class) {
```

Comment:
You can use `$type->getName()` instead of `get_class`.

Reply:
I would rather compare the classes. Similarly to other interfaces’ `getName()` methods, this one may be deprecated and removed in the future.